### ISSUE TITLE
Повышение качества моделей 

### ISSUE THREAD

**Issue by evgpat** [evgpat/in_one_click#2]
- Градиентный бустинг (Gradient Boosting): sklearn.ensemble.GradientBoostingClassifier
- XGBoost: xgboost.XGBClassifier
- CatBoost: catboost.CatBoostClassifier
